guides.md

```diff
@@ -127,6 +128,16 @@ Instructions:
 [Chrome](https://www.techrepublic.com/article/pro-tip-add-custom-search-engines-in-chrome-for-more-efficient-searching/)
 | [Firefox (requires extension)](https://addons.mozilla.org/en-US/firefox/addon/add-custom-search-engine/)
 
+## Storage: Sensitive Data
+
+In general, most staff prefer not to use third-party SaaS services, like Google Drive, for storing sensitive information.
+
+We store **short strings** (like SINs or codes) as individual credentials in Passbolt. These can be shared with both the providing individual and a privileged group that requires access, like Finance WG.
+
+If we must **store sensitive docs** (like PDFs) on GDrive for convenience, we first encrypt them with a password. We keep this in Passbolt as entry "GDrive: File Encryption". All employees can access this password. Simpler encryption schemes are preferred, for example, default PDF encryption.
```

Comment:
Should we use generic `Drive` instead of `GDrive` in case we change the Drive solution in the future?

Reply:
```suggestion
If we must **store sensitive docs** (like PDFs) in a shared drive for convenience (e.g., Google Drive),
we first encrypt them with a password.
We keep this in Passbolt under the entry "Shared Drive: File Encryption".
All employees can access this password.
Simpler encryption schemes are preferred, for example, default PDF encryption.
```

this work?

Reply:
Maybe we should recommend `some-file.encrypted.pdf` or `some-file.enc.pdf`